.github/workflows/build.yml

```diff
@@ -64,18 +64,18 @@ jobs:
 
       - name: gclient sync
         run: |
-          gclient-prepare-sync.sh --reduce-deps --shallow-sync
+          src/flutter/ci/tizen/gclient-prepare-sync.sh --reduce-deps --shallow-sync
           gclient sync -v --no-history --shallow
 
       - name: build
         run: |
-          cache-checksum.sh restore src/out/$OUTPUT_NAME
-          build-engine.sh \
+          src/flutter/ci/tizen/cache-checksum.sh restore src/out/$OUTPUT_NAME
+          src/flutter/ci/tizen/build-engine.sh \
             --target-os ${{ matrix.os }} \
             --target-arch ${{ matrix.arch }} \
             --target-triple ${{ matrix.triple }} \
             --runtime-mode ${{ matrix.mode }} $BUILD_TARGET_OPT
-          cache-checksum.sh save src/out/$OUTPUT_NAME
+          src/flutter/ci/tizen/cache-checksum.sh save src/out/$OUTPUT_NAME
```

Comment:
How about using the `working-directory` keyword and changing relative paths inside `build-engine.sh` to paths relative to the script itself?

Reply:
I will fix this part after the `Dockerfile` of this PR is merged. I will add `src/flutter/ci/tizen` to `GITHUB_PATH` in this workflow.